tests/test-sources/plugins/completion/cmp-all-sources.nix

```diff
@@ -0,0 +1,66 @@
+{pkgs, ...}: {
+  all-sources = {
+    plugins = {
+      copilot-lua = {
+        enable = true;
+
+        panel.enabled = false;
+        suggestion.enabled = false;
+      };
+
+      cmp = {
+        enable = true;
+        settings.sources =
+          [
```

Comment:
You should be able to do something like:
```nix
settings.sources = let
    knownSources = import ../../../cmp/sources.nix;
    filteredSources =
            pkgs.lib.optional
            (pkgs.stdenv.hostPlatform.system != "aarch64-linux")
            {name = "cmp_tabnine";}
in 
    builtins.filter (src: !(builtins.elem src filteredSources)) (mapAttrsToList (k: _: {name = k;}) knownSources)
```

To avoid needing to list each source

Reply:
Done.